test/unit/vl-mapactions-box-select-action.test.js

```diff
@@ -22,18 +22,17 @@ describe('box select action', () => {
         };
       },
     }, callback);
-    sinon.stub(action.dragBoxInteraction, 'getGeometry').returns({
-      getExtent: () => {},
-    });
+    sinon.stub(action.dragBoxInteraction, 'getGeometry').returns({getExtent: () => {}});
+    action.map = {
+      on: sinon.spy(),
+      un: sinon.spy(),
+      render: sinon.spy(),
+    };
```

Comment:
ik zou oppassen in testen van OL stuff met mocken van OL stuff ...

Reply:
Op unit niveau denk ik dat selectief mocken niet een probleem is, maar we zullen oppassen.

Reply:
Na overleg enkele spy objecten vervangen door een OL map.

Reply:
Issue #36 werd aangemaakt om de andere Mockitis symptomen te bekijken.